controllers/cloud.redhat.com/providers/kafka/appinterface.go

```diff
@@ -40,7 +40,7 @@ func (a *appInterface) Provide(app *crd.ClowdApp, c *config.AppConfig) error {
 			Name:      topic.TopicName,
 		}
 
-		err := validateKafkaTopic(a.Ctx, a.Client, topicName)
+		err := validateKafkaTopic(a.Ctx, a.Client, topicName, app)
```

Comment:
With the above comment, this would make this change here unnecessary

Reply:
Yeah, I definitely think this is a small price to pay for consistency and DRY. I could make this a pointer if you are concerned about memory?

Reply:
See above. I misunderstood what you meant. Removed.